bin/ut-release.js

```diff
@@ -8,7 +9,14 @@ conventionalRecommendedBump({
     if (err) {
         throw err;
     } else {
-        require('../lib/exec')('npm', ['version', result.releaseType, '-m', '[ci-skip][ci skip] version incremented to %s']);
+        var releaseType = result.releaseType;
+        if (
+            (process.env.gitlabSourceBranch && /(^ci_.*)/.test(process.env.gitlabSourceBranch)) ||
+            (process.env.GIT_BRANCH && /(^origin\/ci_.*)/.test(process.env.GIT_BRANCH))
+        ) {
+            releaseType = 'patch';
```

Comment:
This logic is still not enough, as depending on branch, we may need to decide if prerelease version or patch version is needed.
@pierrekande, @psstefanov, @ngile, @stambata, @zetxx 
Let's have a meeting to conclude on versioning strategy of branches.

Meanwhile, please inform yourself about prerelease semantics:
https://github.com/npm/node-semver#prerelease-tags

and also the premajor, preminor, prepatch and prerelease parameters of the command `npm version`, as described here:
https://github.com/npm/node-semver#functions